### PR TITLE
Integration of starter kit

### LINKED ISSUE #23: Integrate starterkit into this site
Eventually the starterkit is going to be a child theme or can be placed here at the moment we are going to rewrite twig components here into WP partials.

- https://github.com/moxie-lean/patternlab-starterkit-twig

### PR DESCRIPTION
**Pull request by mitogh**
This closes #23

### CODE REVIEW

**Comment by mitogh** [comment]
This change just completes the migration from twig templates to PHP, #23.

After the migration has been completed we still need to work in some outstanding component such as: 

- #21 
- #22 
- #20 
- #17 

And other tasks that are created from this change.

**Comment by Rulox** [comment]
Hey @mitogh CI is not working here, could you have a look? thanks !

**Comment by mitogh** [comment]
Changes has been updated @Rulox

File: patterns/webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      {
+        test: /\.scss$/,
+        loader: ExtractTextPlugin.extract( 'style', 'css!sass' )
+      }
+    ]
+  },
+  plugins: [
+    new ExtractTextPlugin( 'css/style.css' )
+  ]
+  // , sassLoader: {
```
Comment (Rulox) [inline patterns/webpack.config.js]: Is this something we can remove? @mitogh

Comment (mitogh) [inline patterns/webpack.config.js]: Yes is removed / updated on following PR.

**Pull request merged**